tests/VolleyManagement.UnitTests/Services/TeamService/TeamServiceTests.cs

```diff
@@ -893,6 +893,125 @@ public void ChangeCaptain_PlayerIsPlayerOfAnotherTeam_ValidateExceptionThrown()
         }
         #endregion
 
+        #region Add and remove players
+        /// <summary>
+        /// Test for AddPlayers() method. 
+        /// </summary>
+        [TestMethod]
+        public void AddPalyers_ToTeam_AllPlayerNew_TeamUpdated()
```

Comment:
1) Typo
2) Test name should consist of 3 parts
3) What does it mean "AllPlayerNew"?